public/scripts/upload.js

```diff
@@ -24,6 +24,7 @@ async function runWorker(worker) {
   worker.postMessage({
     imageUploadList: imageUploadList,
     location: window.location.toString(),
+    deviceId: window.deviceDetail.device_id,
     deviceInfo: window.deviceInfo,
```

Comment:
We don't have device id in `deviceInfo`?

Reply:
@limouren See https://github.com/oursky/mockuphone.com/blob/main/src/pages/model/%5Bmodel%5D.astro#L83

deviceInfo is actually the entire `device_info.json`, which contains all 113 devices

Tried removing this, but it will break something

Question: Do you think it's worth spending the time to remove all such legacy codes when I am working on new feature? IMO currently the technical debt is still bearable, and it's not blocking my development; so I chose not to clean these up